environment/vm/lima/config.go

```diff
@@ -9,9 +10,13 @@ import (
 const configFile = "/etc/colima/colima.json"
 
 func (l limaVM) getConf() map[string]string {
+	log := l.Logger(context.Background())
+
 	obj := map[string]string{}
 	b, err := l.Read(configFile)
 	if err != nil {
+		log.Error(err)
```

Comment:
This is not meant to be displayed to the user, as it is expected to fail silently during initial run.
As you can see the error is also discarded on line 24 below.

If at all it would be displayed, the log level should be trace.